src/main/scala/tech/christopherdavenport/twitterstorm/Resource.scala

```diff
@@ -0,0 +1,47 @@
+package tech.christopherdavenport.twitterstorm
+
+import cats.effect.{Async, Effect}
+import fs2.Stream
+
+abstract class Resource[F[_], A] { self =>
+  def apply[B](f: A => Stream[F, B]): Stream[F, B]
+
+  def serve(implicit F: Async[F]): Stream[F, Nothing] =
```

Comment:
It might not make sense to "serve" anything other than a `Server`.  Useful function, questionable name.